dist/index.html

```diff
@@ -4,7 +4,7 @@
 <head>
   <meta charset="utf-8">
   <title>WHISPers</title>
-  <base href="/">
+  <base href="C:/Program Files/Git/">
```

Comment:
This might be a dumb question but is this supposed to be pointing to something on the C drive?

Reply:
Nope. Thanks. ever since I switched to git bash in VS Code, the normal build command command results in that. Its interpolating `"/"` (location of the git bash exe) instead of taking `"/"` literally